presidio-image-redactor/presidio_image_redactor/dicom_image_redactor_engine.py

```diff
@@ -741,6 +769,7 @@ def _redact_single_dicom_image(
         padding_width: int,
         overwrite: bool,
         dst_parent_dir: str,
+        return_bboxes: bool,
```

Comment:
This is called `return` but it saves it to file. Can we consider renaming it to save?

Reply:
That is a good point. The reason why I used "return_bboxes" here is to have consistency between the argument names in `.redact()`, `.redact_from_file()`, and `.redact_from_directory()`.

Do you suggest we prioritize consistency in the argument names (since these methods are highly similar), or should we adjust argument names as necessary in this group of methods?

Reply:
No problem, I've gone ahead and changed it. Thank you for the suggestion!